### PR TITLE
Add manifest for org.kde.kfloppy

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://webchat.kde.org/#/room/#flatpak:kde.org
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfloppy

File: org.kde.kfloppy.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "org.kde.kfloppy",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfloppy",
+    "rename-icon": "kfloppy",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
```
Comment (hfiguiere) [inline org.kde.kfloppy.json:9]:
```suggestion
    "finish-args": [
        "--share=ipc",
        "--socket=fallback-x11",
        "--socket=wayland",
        "--filesystem=host"
    ],
```

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfloppy

File: org.kde.kfloppy.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "id": "org.kde.kfloppy",
+    "branch": "master",
```
Comment (hfiguiere) [inline org.kde.kfloppy.json:3]:
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfloppy

File: org.kde.kfloppy.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfloppy",
+    "rename-icon": "kfloppy",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
```
Comment (hfiguiere) [inline org.kde.kfloppy.json:12]: do you need full host filesystem access? It's my understand this is meant to format floppies, nothing else.

File: org.kde.kfloppy.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            "name": "kfloppy",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "c98dbb8c326b380bac0b23233c9015716815923f",
+                  "url": "https://invent.kde.org/kde/kfloppy.git" },
+                { "type": "patch",
+                  "path": "appdata.patch"
+                }
```
Comment (hfiguiere) [inline org.kde.kfloppy.json:25]: did you test this application? According to the application description, it uses external commands, but I doubt they are available in the flatpak sandbox.

**Comment by nedrichards** [comment]
@flathub/kde FYI

File: org.kde.kfloppy.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        "--filesystem=host"
+    ],
+
+    "modules": [
+        {
+            "name": "kfloppy",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "c98dbb8c326b380bac0b23233c9015716815923f",
```
Comment (tsdgeos) [inline org.kde.kfloppy.json:21]: you need to use a tarball not a git commit

Comment (flyingcakes85) [reply org.kde.kfloppy.json]: https://github.com/flathub/flathub/pull/2749/commits/fcee4e64bf3fe206590b00ce89fc014ca88c91a4

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfloppy

**Comment by travier** [comment]
Thanks for the work here. I'm afraid that KFloppy is a bad candidate for a Flatpak as it won't have the privileges to actually format Floppy disks. Sorry I should have removed it from the list.

**Comment by flyingcakes85** [comment]
No issues :+1: I did learn something new for sure.
I'm closing this pull request.

**Pull request closed without merging**